server/src/helpers/game/commands/gm/GMCreateGold.ts

```diff
@@ -0,0 +1,23 @@
+import { IMacroCommandArgs, IPlayer } from '../../../../interfaces';
+import { MacroCommand } from '../../../../models/macro';
+
+export class GMCreateGold extends MacroCommand {
+
+  aliases = ['@gold'];
+  isGMCommand = true;
+  canBeInstant = false;
+  canBeFast = false;
+
+  execute(player: IPlayer, args: IMacroCommandArgs) {
+    const gold = this.game.itemCreator.getSimpleItem('Gold Coin');
+    const value = +args.arrayArgs[0];
+    if (!value) {
```

Comment:
This needs to check if(!value || isNaN(value))

Reply:
Thanks, added this and a few other checks. There's still a cute interaction where a hex value is a valid number, but that seems fine I believe